exdb/tests/integration_tests.py

```diff
@@ -579,9 +579,9 @@ def post_data(self, status='pe', invalid_description=False, save=False, author=N
             'start_datetime': e.start_datetime.strftime("%Y-%m-%d %H:%M:%S"),
             'end_datetime': e.end_datetime.strftime("%Y-%m-%d %H:%M:%S"),
             'type': e.type.pk,
-            'subtype': e.subtype.pk,
             'audience': e.audience,
             'attendance': 0,
+            'subtype': [self.create_subtype().pk],
```

Comment:
This should probably be changed to `[st.pk for st in e.subtype.all()]` in case we change how `create_subtype()` works in the future.